### ISSUE TITLE
[OpenAPI] Ability to skip a field using a condition

### ISSUE THREAD

**Issue by Mcfloy** [poem-web/poem#220]
## Description of the feature
The usage of Option in the body results with `null` values in the response if the Option is none, the serde crate allows to skip to serialize if a condition is true. This is done thanks to the [skip_serializing_if](https://serde.rs/attr-skip-serializing.html) with `Option::is_none`.

The new feature would be to add a `#[oai(skip_serializing_if = "Option::is_none")` to hide nullable values from the response.

## Code example (if possible)
```rust
#[derive(Object)]
struct MyResponse {
  #[oai(skip_serializing_if = "Option::is_none")
  optional_filter: Option<String>
}
```

**Comment by sunli829** [comment]
Released in `v1.3.9` 😁

**Comment by Mcfloy** [comment]
🎉 That's perfect! Thank you very much!
